### PR TITLE
Add Unit tests to document current behavior

### PR DESCRIPTION
**Pull request by rbayliss**

### CODE REVIEW

**Comment by rbayliss** [comment]
I'm not really familiar with vows, so let me know if I did this wrong.

**Comment by rbayliss** [comment]
Refs #1

File: package.json [hunk 1]
```diff
@@ -31,7 +31,8 @@
   },
   "devDependencies": {
     "js-beautify": "^1.6.8",
-    "jshint": "^2.9.4"
+    "jshint": "^2.9.4",
+    "vows": "^0.7.0"
```
Comment (macbre) [inline package.json:35]: So let's remove `"mocha": "^3.2.0"` from dependencies

File: test/parsing-test.js [hunk 2]
```diff
@@ -0,0 +1,76 @@
+
```
Comment (macbre) [inline test/parsing-test.js:1]: Let's run the code through `npm run beautify`

**Pull request merged**